### ISSUE TITLE
Deeper content for the Source Control dimension

### ISSUE THREAD

**Issue by andrewdwelch** [PowerPlatformAF/PowerPlatformAF#73]

**Comment by rpothin** [comment]
Few points we should cover here:
- Why source control is important?
- Which solution can you use? Azure DevOps, GitHub...
- Simple VS Complex "application(s)" consideration
- Mono repository VS Multiple repositories approaches 🤔
- Branching strategies - git flow, github flow, trunk based development...
- What should you be considering for the organization of your code repository?
   - ["Folder structure of a Dataverse project" by @BenediktBergmann](https://benediktbergmann.eu/2021/04/27/folder-structure-of-a-dataverse-project/)

**Comment by BenediktBergmann** [comment]
I do agree with your list. In addition, we should name unpacking of canvas apps as well as maybe co authoring.

How do we go ahead?

**Comment by rpothin** [comment]
Good point, we could mention canvas app unpacking because currently you need to manage it by yourself being careful with potential breaking changes 😊

We could perhaps leverage [Task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists) to break the work - one task for each point identified in the previous comments - and then we will be able to assign ourself on these subtasks to work on the content directly in the issues.
The final step will be to merge the content in the wiki page.

What do you think @BenediktBergmann / @andrewdwelch ?

**Comment by BenediktBergmann** [comment]
I think task lists could be a good start.
I am away for some days and don't have my laptop with me. Could you create those?

**Comment by rpothin** [comment]
Sure @BenediktBergmann I can take this action 👍🏼

@andrewdwelch do you have an objection regarding using the [Task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists) feature?
Without an answer, I will move forward with this plan the next time I will work on this topic (next tuesday night).

**Comment by rpothin** [comment]
Task list added to the description of this issue.

Once one of us is ready to work on the content of one of these parts, he can create a sub-issue from the task list and assign it to himself. The content will be added to the issue and then we will be able to merge everything here.

I hope this organization will help us move forward.
